Gemfile

```diff
@@ -23,6 +23,10 @@ gem 'rack-mini-profiler'
 gem 'stamp'
 gem 'sentry-raven'
 
+group :development do
```

Comment:
Moved this out of `development, test` group to specifically `development` as recommended by a warning.